dev-vcs/easygit/easygit-1.7.5.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="eg"
+
+DESCRIPTION="Easy GIT (eg), a frontend for git designed for former cvs and svn users"
+HOMEPAGE="https://www.gnome.org/~newren/eg/"
+SRC_URI="https://www.gnome.org/~newren/${MY_PN}/download/${PV}/${MY_PN} -> ${PF}"
```

Comment:
PF is incorrect here. Revisions are intended for internal Gentoo use, and must not be used for versioning sources.

Reply:
@culot: use `${P}` instead.